tests/MongoDB.EntityFrameworkCore.FunctionalTests/Update/UpdateEntityTests.cs

```diff
@@ -48,6 +48,19 @@ class RealisticEntity
         public DateTime lastModified { get; set; }
     }
 
+    class OwningEntity
```

Comment:
Is this used somewhere?

Reply:
Ah, that was used in a different test that I decided not to add, now removed.